test/parallel/test-btoa-atob.js

```diff
@@ -15,3 +15,16 @@ throws(() => buffer.btoa(), /TypeError/);
 
 strictEqual(atob(' '), '');
 strictEqual(atob('  YW\tJ\njZA=\r= '), 'abcd');
```

Comment:
We should also check for `\f`, probably an overlook from last PR.
```suggestion
strictEqual(atob('  Y\fW\tJ\njZ A=\r= '), 'abcd');
```